README.md

```diff
@@ -62,11 +62,25 @@ You may want to store multiple CSV files in a folder that's inside of Starfish,
 Log in to GitHub and [register a new personal access token](https://github.com/settings/tokens/new) (you can find this under Profile > Settings > Developer Settings > Personal access token > "Generate new token"). Fill the "Note" field with e.g. "Starfish" or another description. You don't need to select any scopes. (By default, a token is allowed read-only access to public information, and that's all Starfish needs). Click "Generate token". Copy the access token and store it as you will need it for the next step.
 
 #### Next, Create a file named .env, copy the contents of the .env.template file into it, and add your values to the new file.
+
 - Paste the access token into GITHUB_TOKEN
-- TIMEZONE allows you to specify when your day begins and ends. This must be a value from the [IANA Time Zone Database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones). Typically, this would either be "Etc/UTC" or the time zone of your organization's main office, such as "America/Los_Angeles". 
+- CSV_COLUMN_NUMBER_FOR_GITHUB_ID and CSV_COLUMN_NUMBER_FOR_ALTERNATE_ID should match the columns in the input file.
+  - The CSV you input will be turned into an array, so the numbers for the CSV columns are zero-indexed.
+  - For example, for the example CSV above, `CSV_COLUMN_NUMBER_FOR_GITHUB_ID = "0"` and `CSV_COLUMN_NUMBER_FOR_ALTERNATE_ID = "1"`
+  - If you choose not to use an alternate id, you can put the same value in both fields
+- TIMEZONE allows you to specify how Starfish should assign issues to days. The default is UTC, which works well for organizations with multiple locations. See the "Time zones" section for details.
```

Comment:
Is issues the correct word here?
I'm not sure what that means - `how Starfish should assign issues to days`. Do you mean contributions? Or different timezones?

Reply:
"Issues" was not the right word. I have changed it to "contributions". I wonder if "allocate" would be slightly better than "assign".

Reply:
As far as I can tell, and I looked a few different places, this still says "issues". If it's changed, could you show me where to look?

Allocate might be better. 

I could also see "allows you to specify which timezone Starfish should use for deciding the day a contribution happened." or something like that.